data/NHNSW_2022/metadata.yml

```diff
@@ -0,0 +1,278 @@
+source:
+  primary:
+    key: NHNSW_2022
+    bibtype: Online
+    year: 2022
+    author: National Herbarium of New South Wales
+    title: The Plant Information Network System of The Royal Botanic Gardens and Domain Trust Version 2.0.
+    url: https://plantnet.rbgsyd.nsw.gov.au/
+contributors:
+  data_collectors:
+  - last_name: Duretto
+    given_name: Marco
+    ORCID: 0000-0003-1013-4291
+    affiliation: National Herbarium of New South Wales, Australian Institute of Botanical
+      Science, Royal Botanic Gardens and Domain Trust, Sydney, Australia
+  - last_name: Coleman
+    given_name: David
+    ORCID: 0000-0002-1773-6597
+    affiliation: Evolution & Ecology Research Centre, School of Biological, Earth,
+      and Environmental Sciences, UNSW Sydney, Australia
+    additional_role: .na
+    notes: Responsible for extracting data from online resource
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      filter(str_detect(taxon_name, " "))
```

Comment:
Curious. This is definitely needed?

Reply:
Yes, because David's scraped data includes the genera and family for "inference filling" and this is the easiest way to get rid of those values.